pkg/model/last_backup_run.go

```diff
@@ -0,0 +1,50 @@
+package model
+
+import (
+	"sync"
+	"time"
+)
+
+// LastBackupRun stores the last run times for both full and incremental backups.
+type LastBackupRun struct {
+	mu sync.RWMutex
+	// Last time the Full backup was performed.
+	Full *time.Time
+	// Last time the Incremental backup was performed.
+	Incremental *time.Time
```

Comment:
Make the fields private to prevent setting them directly. Add getters if required.